bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardCanvas.as

```diff
@@ -107,10 +106,6 @@ package org.bigbluebutton.modules.whiteboard.views {
 			whiteboardToolbar.addEventListener(WhiteboardButtonEvent.ENABLE_WHITEBOARD, onEnableWhiteboardEvent);
 			whiteboardToolbar.addEventListener(WhiteboardButtonEvent.DISABLE_WHITEBOARD, onDisableWhiteboardEvent);
 			
-			var stpl:Listener = new Listener();
-			stpl.type = SwitchedPresenterEvent.SWITCHED_PRESENTER;
-			stpl.method = onSwitchedPresenterEvent;
-			
 			presenterChange(UsersUtil.amIPresenter(), UsersUtil.getPresenterUserID());

```

Comment:
This won't be right either. It's going to default the presenter to the meeting presenter. The whiteboard has to wait to be told the pod presenter by the presentation code.